src/pkg/caendr/caendr/services/indel_primer.py

```diff
@@ -45,13 +27,6 @@
 MIN_SV_SIZE = 50
```

Comment:
can we move these values (MIN_SV_SIZE, MAX_SV_SIZE) under the admin Tool Versions/Settings?

Reply:
Yes, I believe so.  They're only used in this file to set the min/max size that an indel can be to be displayed.

Where would we store them in the database?

Reply:
Good question -- most likely DataStore with a new `kind=settings`, and a single row for the tool. Also, our code needs to create this row with default values (seed values included in the repo). End-users will then be able to change them via UI option in the Tool/Versions/Settings.

Reply:
this also should be a new PR and not included in this release.